deep_qa/training/multi_gpu.py

```diff
@@ -0,0 +1,89 @@
+from keras.layers import merge
+from keras.layers.core import Lambda
+import keras.backend as K
+import tensorflow
+from .models import DeepQaModel
```

Comment:
I think having a blank line separating import groups is helpful (one block for standard python imports, one block for third-party imports, and one block for first-party imports).